system-int-test/test/cmr/system_int_test/search/scrolling_search_test.clj

```diff
@@ -135,6 +135,46 @@
             (is (= (count all-grans) hits))
             (is (data2-core/refs-match? [gran3 gran4] result))))
 
+        (testing "Remaining results returned on last search"
```

Comment:
Since we also support scrolling for collections we should have a test that deferred scrolling works for collections as well as granules.